Tickets/sitcom 1401

JackieS-NL opened the pull request:
**Introduction to GIS** reviewed by Franco Colleone. 

In Safety/Safety-Systems/ 

Origin branch (may have to change) SITCOM-1402

isotuela commented on Safety/Safety-Systems/GIS.rst:
```diff
@@ -1,6 +1,325 @@
+
+* **AcFiEa**: Access Control System / Fire Main Controller / Earthquake Control System. The safety gates to the pier intermediate level and to level 8 are referred as access control system.
+
+* **M2CAM**: M2 area, includes the M2 actuator, the M2 hexapod, the Camera Hexapod and the Camera Rotator.
```
What do you mean by M2 actuator?

JackieS-NL replied:
Is the name given to that section, not sure if it has a meaning, I will ask

JackieS-NL replied:
It refers to the actual actuators of M2

isotuela commented on Safety/Safety-Systems/GIS.rst:
```diff
@@ -1,6 +1,325 @@
+
+* **AcFiEa**: Access Control System / Fire Main Controller / Earthquake Control System. The safety gates to the pier intermediate level and to level 8 are referred as access control system.
+
+* **M2CAM**: M2 area, includes the M2 actuator, the M2 hexapod, the Camera Hexapod and the Camera Rotator.
```
M2 Hexapod starting with capital H as the others?

isotuela commented on Safety/Safety-Systems/GIS.rst:
```diff
@@ -1,6 +1,325 @@
+* **AUX IS**: Auxiliary Telescope Controller IS.
+
+
+You can access the **GIS Detections and Actions** section with :guilabel:`Det-Act` button
```
Finish sentence with a . or :?

isotuela commented on Safety/Safety-Systems/GIS.rst:
```diff
@@ -1,6 +1,325 @@
+
+The status of the system is displayed, the symbols and their representation are: 
+
+* BYP within an orange square: Indicates the action is bypassed. It logs alarms but doesn't trigger any action.
```
Perhaps highlight BYP in bold or italics?

isotuela commented on Safety/Safety-Systems/GIS.rst:
```diff
@@ -1,6 +1,325 @@
+    E.g M2 interlock can be reset in GIS.
+
+* **Network based systems** reset on their own in GIS main panel after the reset in done in the specific subsytem.
+     E.g. M1M3 interlock should be reset in M1M3 IS and then it will be clear in GIS main panel.
```
Below you have M1M3IS (together), I think that's how it's written.

isotuela commented on Safety/Safety-Systems/GIS.rst:
```diff
@@ -1,6 +1,325 @@
+* **Wired subsystems** are reset in GIS Main Panel. 
+    E.g M2 interlock can be reset in GIS.
+
+* **Network based systems** reset on their own in GIS main panel after the reset in done in the specific subsytem.
```
Consider replacing "after" by "once"

isotuela commented on Safety/Safety-Systems/GIS.rst:
```diff
@@ -1,6 +1,325 @@
+
+.. note::
+    
+    D4 GIS E-stop: Safety gates E-stops are hit very often, so there were physically bypassed. This will be undone when operations start.
```
"so **they** are currently physically bypassed"

isotuela commented on Safety/Safety-Systems/GIS.rst:
```diff
@@ -1,6 +1,325 @@
+
+    * **Network**: TMA, Dome, M1M3, AuxTel.
+
+    * **Signals wires**: Earthquake, Access control, Pflow lift, Calibration Laser, and the top-end subsystems (M2Cam, these are M2, M2Hex, Camera Rotator, Camera Hexapod).
```
"M2Cam that includes M2..."

isotuela commented on Safety/Safety-Systems/GIS.rst:
```diff
@@ -1,6 +1,325 @@
+    * To open a gate: Press :guilabel:`release` button (gate is open), then move the handle.
+    * To close a gate: move the handle to close the gate. Press the :guilabel:`lock` button (the gate is closed).
+
+.. admonition:: A note on M1M3 IS interface
```
Check wether M1M3IS is written together or two separate words.

JackieS-NL replied:
I read it as M1M3 IS, fixed.

isotuela commented on Safety/Safety-Systems/GIS.rst:
```diff
@@ -1,6 +1,325 @@
+.. admonition:: Safety gates operation
+
+    * To open a gate: Press :guilabel:`release` button (gate is open), then move the handle.
+    * To close a gate: move the handle to close the gate. Press the :guilabel:`lock` button (the gate is closed).
```
You may capitalize Move here as it's a complete sentence.

JackieS-NL replied:
Fixed, I also capitalized the buttons Release and Lock as in the AuxTel gate are written as such.

isotuela reviewed (changes requested):
Hi Jacqueline! 
Excellent work. Thank you for putting all this together. 

The structure is great and organization is clear. I am only missing instruction on how to reset from the GIS panel, for instance, D4. I can't see the RESET button in the main panel, don't you have to navigate to the Detection Page? Could you please add that information and a screenshot, perhaps in the "Bypass and Reset Actions" section?  

I have also added some minor suggestions for your consideration. 

We shall also merge this into `main` directly, for that, you'll need to rebase, and squash your commits. Let me know if you need help. 

Thank you so much!

JackieS-NL commented:
Ioana, I made your corrections. I added an image of the M2CAM window in the GIS for resetting the E-stop (is under the Top-End-Subsystem title). I improved the text and added a section title for the table. Fixed some labels too.

I squashed my commits and I would like your help doing the rebase.

isotuela commented on Safety/Safety-Systems/GIS.rst:
```diff
@@ -1,6 +1,342 @@
+
+Top End systems** (M2CAM), are a special case. The status listed does NOT indicate the status of each individual element, but rather the **status of the E-Stop circuit** (including reset action) for each element.
+
+After an **STO or power cycle** of the Top End systems, you must :guilabel:`RESET` the GIS E-stop status buttons 
```
I think I am confused with the E-stop status notation as I don´t see it in the screen. Do you mean the M2 ACTUATOR, M2 HEXAPOD... RESET buttons?

isotuela reviewed (changes requested):
Thank you Jacqueline! 
I think I am still confused about the TEA section, with the screenshot attached (where I can´t see the E-stop) and the mentions to E-stop. 
Also, I think we must clearly include how to RESET the Wired Subsystems, even if obvious. 
Thank you so much for all your work!
One small note: Rebasing creates an additional commit, so it´s better to squash at the end, so you don´t have to squash twice.

JackieS-NL commented on Safety/Safety-Systems/GIS.rst:
```diff
@@ -1,6 +1,342 @@
+
+Top End systems** (M2CAM), are a special case. The status listed does NOT indicate the status of each individual element, but rather the **status of the E-Stop circuit** (including reset action) for each element.
+
+After an **STO or power cycle** of the Top End systems, you must :guilabel:`RESET` the GIS E-stop status buttons 
```
That is the one I'm referring, the text was not clear. This is the modified version. 

"After an **STO or power cycle** of the Top End systems, you must :guilabel:`RESET` the GIS E-stop status buttons in the M2CAM window, shown in the figure below, in order to enable the system. 
"

JackieS-NL replied:
for the records, instructions were added with images for reference

JackieS-NL commented on Safety/Safety-Systems/GIS.rst:
```diff
@@ -1,6 +1,342 @@
+
+* Any user can :guilabel:`RESET` a subsystem.
+
+* **Wired subsystems** are reset in GIS Main Panel. 
```
I don't understand, do you mean near the left edge of the window?

In the M2CAM image under Top End System title, it is shown the reset button.

JackieS-NL commented on Safety/Safety-Systems/GIS.rst:
```diff
@@ -69,9 +69,10 @@ The Main Controller manages all other cabinets, and user interaction is primaril
     * **Network**: TMA, Dome, M1M3, AuxTel.
 
-    * **Signals wires**: Earthquake, Access control, Pflow lift, Calibration Laser, and the top-end subsystems (M2Cam that includes M2, M2Hex, Camera Rotator, Camera Hexapod).
+    * **Signals wires**: Earthquake, Access control, Pflow lift, Calibration Laser, and the top-end subsystems: M2Cam (that includes M2, M2Hex, Camera Rotator, Camera Hexapod).
```
agreed, done

JackieS-NL commented on Safety/Safety-Systems/GIS.rst:
```diff
@@ -178,76 +179,106 @@ Some area names are straightforward, while others are:
       :width: 700px
 
-      GIS Detections and actions
+      GIS Detections and Actions pane
```
I changed 'pane' to 'window'.

JackieS-NL commented:
All corrections done, I rebased with main and I squashed the commits into one.

isotuela reviewed (approved):
Thank you for all the changes! 
Ioana
